specs/applicationdata/ApplicationData.md

```diff
@@ -0,0 +1,428 @@
+WinRT APIs they include some patterns out of step with current API practices and recommendations.
+Windows App SDK offers the opportunity to provide a new generation of APIs in line with the latest
+patterns and recommendations for an improved developer experience as well as functional enhancements
+and improved runtime efficiencies.
+
+Microsoft-internal tasks [22308619](https://task.ms/22308619),
+[30002314](https://task.ms/30002314), [39401257](https://task.ms/39401257),
+[39414843](https://task.ms/39414843), [40050113](https://task.ms/40050113) and
+[48332357](https://task.ms/48332357).
+
+# 3. Description
+
+This API supports app data stores focusing on the following scenarios:
+
+* ApplicationData and ApplicationDataManager functionality
+* *Path properties
+* Reliable access
+* Per-Machine data store
+* Unpackaged app data stores
+
+## 3.1. ApplicationData and ApplicationDataManager functionality
+
+This API provides an equivalent superset of
+[Windows.Storage.ApplicationData](https://learn.microsoft.com/uwp/api/windows.storage.applicationdata),
+with the following notable differences:
+
+* **No roaming functionality** -- We don't provide equivalents to Windows.Storage.ApplicationData's
+      [RoamingSettings](https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.roamingsettings),
+      [RoamingStorageQuota](https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.roamingstoragequota),
+      [Version](https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.version),
+      [SetVersionAsync()](https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.setversionasync),
+      [SignalDataChanged()](https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.signaldatachanged)
+      and
+      [DataChanged](https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.datachanged). See [5.1. Platform Support](#51-platform-support) for more details.
+* **GetDefault() works for all packaged apps** -- Equivalent to .Current but works for all packaged
+  apps (not just callers running in an AppContainer). See
+  [3.3. Reliable access](#33-reliable-access).
+* **`GetForPackageFamily()`** -- Equivalent to Windows.Storage.ApplicationDataManager.CreateForPackageFamily().
+* **`GetForUser()`** -- Synchronous equivalent to GetForUserAsync().
+* **`GetForUnpackaged()`** -- Equivalent functionality for unpackaged apps. See
+  [3.5. Unpackaged app data stores](#35-unpackaged-app-data-stores).
+
+## 3.2. *Path properties
+
+New *Path equivalents to Windows.Storage.ApplicationData's *Folder but returning a `String` rather than `StorageFolder`.
+
+## 3.3. Reliable access
+
+`Windows.Storage.ApplicationData.Current` only works for packaged apps **running in an AppContainer**. Processes not running in an AppContainer (e.g. MediumIL) must use the equivalent `Windows.Management.Core.ApplicationDataManager.CreateForPackageFamily(Windows.ApplicationModel.Package.Current.Id.FamilyName)`.
```

Comment:
Please use a tool like `prettier` or your favorite markdown editing package to format the document and word-wrap at about 100 characters.

Reply:
Sorry about that. Thanks for the reminder. Fixed.